pkg/scheduler/algorithm/priorities/resource_allocation.go

```diff
@@ -30,10 +31,14 @@ import (
 
 // ResourceAllocationPriority contains information to calculate resource allocation priority.
 type ResourceAllocationPriority struct {
-	Name   string
-	scorer func(requested, allocable *schedulernodeinfo.Resource, includeVolumes bool, requestedVolumes int, allocatableVolumes int) int64
+	Name      string
+	scorer    func(requested, allocable map[v1.ResourceName]int64, includeVolumes bool, requestedVolumes int, allocatableVolumes int) int64
```

Comment:
define a new type for map[v1.ResourceName]int64 and name it ResourceToValue map so that we can distinguish between the weight map and the value map.

Reply:
👍